### PR TITLE
Docs' improvements

### PR DESCRIPTION
**Pull request by bhavya3024**
1. Add info for in order to run hugo, go is also required
2. Need for append port in url otherwise it doesn't work. (more info in PR comments).
@lizconlan @simonrw @fictorial @SimonWallner

### CODE REVIEW

File: content/en/getting-started/quickstart/index.md [hunk 1]
```diff
@@ -266,7 +266,7 @@ Save these URLs for later use in the sample application.
 
 ### Run the sample AWS application
 
-To access the application, go to [**https://webapp.s3-website.localhost.localstack.cloud/**](https://webapp.s3-website.localhost.localstack.cloud/) in your browser.
+To access the application, go to [**https://webapp.s3-website.localhost.localstack.cloud:4566**](https://webapp.s3-website.localhost.localstack.cloud) in your browser.
```
Comment (bhavya3024) [inline content/en/getting-started/quickstart/index.md:269]: It was giving me connection refused without the PORT `4566`.

File: README.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ or:
 ### Install Hugo
 LocalStack Docs is based on the [Hugo static site generator](https://gohugo.io).
 
-In order to contribute to LocalStack Docs, you need to [install Hugo](https://gohugo.io/getting-started/installing) in order to verify your changes. Make sure to install the _extended_ version of Hugo.
+In order to contribute to LocalStack Docs, you need to [install Hugo](https://gohugo.io/getting-started/installing) in order to verify your changes. Make sure to install the _extended_ version of Hugo. You also need to make sure that `go` is installed in order to run hugo scripts.
```
Comment (bhavya3024) [inline README.md:36]: go is required to run `Hugo serve`

**Review by simonrw — APPROVED** [review]
Hi @bhavya3024, thanks for this contribution! These changes look great, just a couple of suggestions but nothing significant. Happy for you to merge once you've made the changes :tada:

File: content/en/getting-started/quickstart/index.md [hunk 3]
```diff
@@ -323,4 +323,4 @@ Congratulations on deploying an AWS application locally using LocalStack! To exp
 - [Tutorials]({{< ref "tutorials" >}}): Check out our tutorials to learn how to use LocalStack across various AWS services and application stacks.
 - [User Guide]({{< ref "user-guide" >}}): Explore LocalStack's emulated AWS services, third-party integrations, tooling, CI service providers, and more in our User Guide.
 - [References]({{< ref "references" >}}): Dive into LocalStack references for a comprehensive understanding of key internals and technical details.
-- [Blog](https://blog.localstack.cloud): Read our blog posts about LocalStack and the latest enhancements for a better local development and testing experience.
\ No newline at end of file
+- [Blog](https://blog.localstack.cloud): Read our blog posts about LocalStack and the latest enhancements for a better local development and testing experience.
```
[the remark this answers is not in the document]
Comment (bhavya3024) [reply content/en/getting-started/quickstart/index.md:326]: Let me try this

**Comment by bhavya3024** [comment]
@simonrw I have done the required changes! thanks for the review!

**Review by simonrw — COMMENTED** [review]
Awesome thank you for this contribution!

**Pull request merged**